apps/client/.turbo/turbo-dev.log

```diff
@@ -1,162 +0,0 @@
-
```

Comment:
hmm, this file should be ignored,  I thought I added it to the gitignore, but maybe you used this before syncing with my code?